ghost/core/core/server/data/schema/schema.js

```diff
@@ -600,6 +600,45 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    subscriptions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        type: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['free', 'comped', 'paid']]
+            }
+        },
+        status: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['active', 'expired', 'cancelled']]
+            }
+        },
+        member_id: {type: 'string', maxlength: 24, nullable: false, unique: false, references: 'members.id', cascadeDelete: true},
+        tier_id: {type: 'string', maxlength: 24, nullable: true, unique: false, references: 'products.id'},
```

Comment:
Used `tier_id` instead of `product_id` here. The thinking behind the decision is that we'd have less work to do (and less execution time) once we rename the `products` table to `tiers`